types/chromecast-caf-receiver/cast.framework.messages.d.ts

```diff
@@ -2140,35 +2174,40 @@ export class CustomCommandRequestData extends RequestData {
  */
 export class CloudMediaStatus extends MediaStatus {}
 
+/**
+ * Represents current status of break.
+ *
+ * @see https://developers.google.com/cast/docs/reference/caf_receiver/cast.framework.messages.BreakStatus
+ */
 export class BreakStatus {
-    constructor(currentBreakTime: number, currentBreakClipTime: number);
+    constructor(currentBreakTime: number | undefined, currentBreakClipTime: number | undefined);
```

Comment:
Add `?` to both parameters and remove `| undefined`. That way we can omit the arguments if they are undefined while we can still pass undefined to them.

See this playground

https://www.typescriptlang.org/play?#code/GYVwdgxgLglg9mABKMAKAhgfgFyLCAWwCMBTAJ0QB9FwATE4GME2gSkQG8AoRRMkqCDJJ0Abi4BfLilStxMgKxzp4VHQZMWrLisiwEyEBhx5CpMu269+g4YjGSVs+UaUu1Yeo2ZsdKaPBIwGjouPjE5FQ0nho+ljx8AkIi4lLSaMrBqG7pHl6abKJAA

Reply:
I checked the caf_receiver sdk and in fact  `new cast.framework.messages.BreakStatus()` works. I've updated the declaration (I've kept `| undefined` as per TS4.4)